README.md

```diff
@@ -35,6 +60,7 @@ Installing
 Wesnoth can be played on Windows, macOS, and Linux, and can be installed using:
 * [Steam](https://store.steampowered.com/app/599390/Battle_for_Wesnoth/) (all)
 * [itch.io](https://wesnoth.itch.io/battle-for-wesnoth) (Windows and macOS)
+    * Download folder, unzip, then run/open wesnoth.exe
```

Comment:
This is accurate for the standalone download for Windows, however it's incorrect for macOS as well as when played through the itch.io client.